corehq/apps/userreports/models.py

```diff
@@ -96,8 +96,14 @@ class SQLPartition(DocumentSchema):
     constraint = StringProperty()
 
 
+class CitusConfig(DocumentSchema):
+    distribution_type = StringProperty(choices=['reference', 'hash'])
+    distribution_column = StringProperty()
+
+
 class SQLSettings(DocumentSchema):
     partition_config = SchemaListProperty(SQLPartition)
+    citus_config = SchemaProperty(CitusConfig)
```

Comment:
what happens if both configs are defined? Do we want to support that?

Reply:
good question, at present they would both be applied but it would fail since citus only supports partitioning with native PG partition syntax.

I think for now it should be disabled.